modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/IgniteH2Indexing.java

```diff
@@ -1516,6 +1526,9 @@ private void createSqlFunctions(String schema, Class<?>[] clss) throws IgniteChe
 
     /** {@inheritDoc} */
     @Override public void registerCache(CacheConfiguration<?,?> ccfg) throws IgniteCheckedException {
+        space2schema.put(ccfg.getName(), ccfg.getSqlSchema() == null ? '\"' + ccfg.getName() + '\"' : ccfg.getSqlSchema());
+        schema2space.put(ccfg.getSqlSchema() == null ? '\"' + ccfg.getName() + '\"' : ccfg.getSqlSchema(), ccfg.getName());
```

Comment:
The code above looks cryptic to me. Lets calculate schema name once, assign it to a variable and put everywhere we need to.